examples/TestDriver/App.js

```diff
@@ -15,7 +15,10 @@ export default class App extends React.Component {
   render() {
     return (
       <Provider store={store}>
-        <TopNavigator ref={NavigationService.setTopLevelNavigator} />
+        <TopNavigator
+          ref={NavigationService.setTopLevelNavigator}
+          onNavigationStateChange={() => {}} // Checks that manually instrumenting screen tracking works.
```

Comment:
It might be good to expand on this comment, since the context will get lost pretty quickly - i.e. why does having a NOOP function there check what it checks?